cmake/std/atdm/ats1/environment.sh

```diff
@@ -83,50 +68,15 @@ export ATDM_CONFIG_SBATCH_EXTRA_ARGS="$ATDM_CONFIG_SBATCH_EXTRA_ARGS --exclude=n
 export OMP_NUM_THREADS=2
 
 # Common modules and paths
-module load cmake/3.14.6
-module load git/2.19.1
+module load sparc-cmake/3.18.1
```

Comment:
Let's just remove this `module load sparc-cmake/318.1` and just use what is loaded by the `sparc-dev` module which is:

```
$ ssh mutrino

$ which cmake
/usr/bin/cmake

$ cmake --version
cmake version 3.10.2

$ module load sparc-dev/intel-19.0.4_mpich-7.7.15_hsw

$ which cmake
/usr/projects/sparc/tools/cmake/3.18.1/x86_64/bin/cmake

$ cmake --version
cmake version 3.18.1
```